src/vs/editor/common/controller/cursor.ts

```diff
@@ -944,6 +944,7 @@ export class Cursor extends EventEmitter {
 
 		this._handlers[H.JumpToBracket] =				(ctx) => this._jumpToBracket(ctx);
 
+		this._handlers[H.CursorMove] = 					(ctx) => this._move(false, ctx);
```

Comment:
Please rename the method `_move` to `_cursorMove` to easily find it and not get confused with the previous moveTo